cli/cook/http.py

```diff
@@ -29,6 +31,7 @@ def configure(config):
     http_adapter = adapters_module.HTTPAdapter(max_retries=retries)
     session = session_module.Session()
     session.mount('http://', http_adapter)
+    session.headers['User-Agent'] += f' (cs/{cook.version.VERSION})'
```

Comment:
Name changed: `CookCLI` → `cs`

I also decided that it made more sense to append the extra user agent info to the default user-agent header of the response library. Now it looks like this in the logs: `"python-requests/2.18.4 (cs/0.7.1)"`

Reply:
Can we change the order to `cs/0.7.1 (python-requests/2.18.4)`? This makes more sense to me as `cs` is what is more important, that it uses `python-request` is an implementation detail.

Reply:
Based on https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent , `cs` is our product, `0.7.1` is the version and `(python-requests/2.18.4)` is system information.

Reply:
OK, you convinced me. I'll reverse it.